coalib/bearlib/languages/documentation/DocstyleDefinition.py

```diff
@@ -186,13 +238,41 @@ def load(cls, language: str, docstyle: str, coalang_dir=None):
         metadata = cls.Metadata(*(str(docstyle_settings.get(req_setting, ''))
                                   for req_setting in metadata_settings))
 
+        try:
+            docstyle_settings['class_padding']
+            class_padding = cls.ClassPadding(
+                *(int(padding) for padding in tuple(
+                    docstyle_settings.get('class_padding', ''))))
+        except IndexError:
+            class_padding = cls.ClassPadding('', '')
+
+        try:
+            docstyle_settings['function_padding']
```

Comment:
yup we can remove this I guess...
as now I will use `docstyle_settings['function_padding']` instead of `docstyle_settings.get()`
which will throw a `KeyError` :+1: